tools/topology/m4/tdfb_coef_flat.m4

```diff
@@ -0,0 +1,19 @@
+# Exported EQ 09-Sep-2020
```

Comment:
There's quite many of these and more to come later. I think I should rename these to m4/tdfb/coef_flat.m4 etc. to avoid clutter.

Reply:
This move is now done.